app/routes.py

```diff
@@ -11,22 +13,44 @@ def index():
         {"author": {"username": "Sal"}, "body": "Beautiful day in Portland!"},
         {"author": {"username": "Michael"}, "body": "Pyrates are so cool!"},
         {"author": {"username": "Scott"}, "body": "Ooh, Bootstrap!"},
+        {"author": {"username": "Allyson"}, "body": "Hello, World!"},
     ]
     return render_template("index.html", title="Home", user=user, posts=posts)
 
 
 @app.route("/login", methods=["GET", "POST"])
 def login():
-    form = LoginForm()
+    if current_user.is_authenticated:
+        return redirect(url_for('index'))
+    form=LoginForm()
     if form.validate_on_submit():
-        flash(
-            f"Login requested for user {form.username.data}, "
-            f"remember_me={form.remember_me.data}"
-        )
-        return redirect(url_for("index"))
-    return render_template("login.html", title="Log In", form=form)
+        user=User.query.filter_by(username=form.username.data).first()
```

Comment:
The login page breaks when I click the login button. The variable User that is being used in this line is not defined. See this error:

<img width="742" alt="image" src="https://user-images.githubusercontent.com/25067109/168739780-174d727b-ae41-4621-b2da-7c38fb4a0111.png">

Reply:
fix applied, still getting the error.